pkgs/hover/fix-build-docker.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/cmd/build.go b/cmd/build.go
+index 2853826..d78cc6b 100644
+--- a/cmd/build.go
++++ b/cmd/build.go
+@@ -42,7 +42,7 @@ func initCompileFlags(cmd *cobra.Command) {
+ 	cmd.PersistentFlags().StringVar(&buildOrRunCachePath, "cache-path", enginecache.DefaultCachePath(), "The path that hover uses to cache dependencies such as the Flutter engine .so/.dll")
+ 	cmd.PersistentFlags().StringVar(&buildOrRunOpenGlVersion, "opengl", config.BuildOpenGlVersionDefault, "The OpenGL version specified here is only relevant for external texture plugin (i.e. video_plugin).\nIf 'none' is provided, texture won't be supported. Note: the Flutter Engine still needs a OpenGL compatible context.")
+ 	cmd.PersistentFlags().StringVar(&buildOrRunEngineVersion, "engine-version", config.BuildEngineDefault, "The flutter engine version to use.")
+-	cmd.PersistentFlags().BoolVar(&buildOrRunDocker, "docker", false, "Execute the go build and packaging in a docker container. The Flutter build is always run locally")
++	cmd.PersistentFlags().BoolVar(&buildOrRunDocker, "docker", true, "Execute the go build and packaging in a docker container. The Flutter build is always run locally")
+ 
+ 	cmd.PersistentFlags().MarkHidden("branch")
+ }
+diff --git a/cmd/docker.go b/cmd/docker.go
+index 7727533..d488224 100644
+--- a/cmd/docker.go
++++ b/cmd/docker.go
+@@ -50,6 +50,7 @@ func dockerHoverBuild(targetOS string, packagingTask packaging.Task, buildFlags
+ 		"--mount", "type=bind,source=" + engineCacheDir + ",target=/root/.cache/hover/engine",
+ 		"--mount", "type=bind,source=" + dockerGoCacheDir + ",target=/go-cache",
+ 		"--env", "GOCACHE=/go-cache",
++		"--env", "CGO_ENABLED=1",
```

Comment:
The flag to enable static builds is `CGO_ENABLED=0`: https://medium.com/@diogok/on-golang-static-binaries-cross-compiling-and-plugins-1aed33499671

But in this specific case I don't know if this would work, since `go-flutter-desktop` use libraries that depends on C: https://github.com/go-flutter-desktop/go-flutter/blob/master/go.mod#L8-L9.

I am actually quite surprised that NixOS `unstable` is building this fully static.

Reply:
Even setting `CGO_ENABLED=0` didn't change anything.

Yeah, I am quite surprised this works in `unstable`, maybe it is some recent changes, maybe it is because it uses `go` `1.15`.

Anyway, going to approve here, just delete this line first since it doesn't make any difference.